tags/tag-security-and-compliance/README.md

```diff
@@ -12,9 +12,9 @@ Security hygiene, Policy-as-code, Compliance, Auditing, Threat Modeling, Secure
 - Evan Anderson (**[@evankanderson](https://github.com/evankanderson)**), Custcodian (Term: 2025-07-01 - 2026-06-30)
 - Marina Moore (**[@mnm678](https://github.com/mnm678)**), Edera (Term: 2025-07-01 - 2027-06-30)
 ### Tech Leads
-- Brandt Keller (**[@brandtkeller](https://github.com/brandtkeller)**), Defense Unicorns (Term: 2025-07-02 - 2026-06-30)
+- Brand Keller (**[@brandkeller](https://github.com/brandkeller)**) (Term: 2025-07-02 - 2026-06-30)
```

Comment:
Is this intentionally being overwritten? The previous version is correct, but I'm not sure if this should be done elsewhere for the automation

Reply:
@brandtkeller for laughs ^